tests/Integration/tests/markdown/expected/README.html

```diff
@@ -0,0 +1,279 @@
+                </nav>
+            </div>
+            <div class="page-main-content">
+                <div class="rst-content">                        <div aria-label="breadcrumbs navigation" class="breadcrumb-bar" role="navigation">
+        
+<ol class="breadcrumb">
+                <li aria-current="page"  class="breadcrumb-item  active">Child&#039;s Swing Usage Guide</li>
+        </ol>
+
+        <div class="breadcrumb-additions">
+            <a class="btn btn-sm btn-light" href="_sources/Index.rst.txt" rel="nofollow noopener" target="_blank">
+                <span class="btn-icon"><span class="fas fa-code"></span></span>
+                <span class="btn-text">View source</span>
+            </a>
+            <a class="btn btn-sm btn-light" href="https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingDocsOfficial/GithubMethod.html" id="btnHowToEdit" rel="nofollow noopener" target="_blank">
+                <span class="btn-icon"><span class="fas fa-info-circle"></span></span>
+                <span class="btn-text">How to edit</span>
+            </a>
+            <a class="btn btn-sm btn-primary" href="https://github.com/TYPO3-Documentation/DocsTypo3Org-Homepage/edit/main/Documentation/Index.rst" id="btnEditOnGitHub" rel="nofollow noopener" target="_blank">
+                <span class="btn-icon"><span class="fab fa-github"></span></span>
+                <span class="btn-text">Edit on GitHub</span>
+            </a>
+        </div>
+    </div>
+                    <div class="document" itemscope="itemscope" itemtype="http://schema.org/Article" role="main">
+                        <div itemprop="articleBody">
+                            <!-- body -->
+                                <h1>Child&#039;s Swing Usage Guide</h1>
+
+<p>Welcome to the world of fun and laughter with our child&#039;s swing! Follow thesimple steps below to ensure a safe and enjoyable experience for your littleone.</p>
+<h2>Table of Contents</h2>
+
+
+
+<ul>
+    <li class="dash"><p><a href="#installation">Installation</a></p></li>
+
+    <li class="dash"><p><a href="#usage">Usage</a></p></li>
+
+    <li class="dash"><p><a href="#safety-guidelines">Safety Guidelines</a></p></li>
+
+    <li class="dash"><p><a href="#maintenance">Maintenance</a></p></li>
+
+    <li class="dash"><p><a href="#troubleshooting">Troubleshooting</a></p></li>
+
+</ul>
+
+<hr />
+
+<h2>Installation</h2>
+
+<p><img src="https://upload.wikimedia.org/wikipedia/commons/a/a3/Forgiveness_5.gif" alt="Swing Image"/></p>
+
+    
+<ol>
+    <li><p><strong>Choose a Sturdy Location:</strong> Select a location with a sturdy beam or swingset structure capable of supporting the weight of the swing and the child.</p></li>
+
+    <li><p><strong>Securely Attach the Swing:</strong> Use strong and reliable <a href="https://example.com/ropes">ropes</a>or chains to securely attach the swing to the chosen location. Double-checkthe knots or hardware to ensure they are tight and safe.</p></li>
```

Comment:
A space must be inserted after the word "ropes", as the output of the code displays "ropesor" at this point, so a small typo

Reply:
This seems to be standard behaviour of Markdown and can be fixed by putting a &nbsp; here or two empty spaces at the end of the line (which do however get removed by IDEs unfortunary)

Reply:
I solved it like that now in the markdown. good catch

Reply:
I see. Good to know^^